### PR TITLE
WFLY-13496 Change the packaging of MP Health quickstart to ejb to ena…

### PR DESCRIPTION
**Pull request by xstefank**
…ble deployment through IDEs

https://issues.redhat.com/browse/WFLY-13496

### CODE REVIEW

File: microprofile-health/pom.xml [hunk 1]
```diff
@@ -13,7 +13,7 @@
   </parent>
 
   <artifactId>microprofile-health</artifactId>
-  <packaging>jar</packaging>
+  <packaging>ejb</packaging>
```
Comment (emmartins) [inline microprofile-health/pom.xml:16]: with this documentation change shouldn't the pom packaging stay as jar?

Comment (emmartins) [reply microprofile-health/pom.xml:16]: ok, think I understand now, the generated pom.xml has no such change, right?

Comment (xstefank) [reply microprofile-health/pom.xml:16]: There is no way to generate the packaging directly as ejb, so users for step-by-step instructions need to do the change themselves.

This change is required as the QS itself is consumed in the JB studio. They don't follow step-by-step AFAIK.

File: microprofile-health/README.adoc [hunk 2]
```diff
@@ -97,7 +97,18 @@ Open the project in your favourite IDE.
 
 Open the generated `pom.xml`.
 
-The first thing to do is to setup our dependencies. Add the following section to your
+The first thing to do is to change the packaging to `ejb` as this is required by
```
Comment (emmartins) [inline microprofile-health/README.adoc:100]: please change `ejb` to `war`

File: microprofile-health/pom.xml (hunk 1 above)
Comment (emmartins) [reply microprofile-health/pom.xml:16]: please change to war

**Pull request merged**